lib/include/tick/preprocessing/longitudinal_features_lagger_mp.h

```diff
@@ -0,0 +1,228 @@
+//

```

Comment:
Why don't you put CooMatrix in a separate file?

Reply:
move to ```array/coo_matrix.h```